lxd/storage/backend_lxd.go

```diff
@@ -1828,7 +1828,7 @@ func (b *lxdBackend) imageConversionFiller(imgPath string, imgFormat string) fun
 		cmd := []string{
 			// Run with low priority to reduce CPU impact on other processes.
 			"nice", "-n19",
-			"qemu-img", "convert", "-f", imgFormat, "-O", "raw", imgPath, diskPath,
+			"qemu-img", "convert", "-f", imgFormat, "-O", "raw", imgPath, diskPath, "-t", "writeback",
```

Comment:
Can you attribute this to https://github.com/lxc/incus/pull/1015/commits/bbc1ea8cd0cb01b4830fd960007f4c5b6d05edf2 please by way of a cherry-pick

Reply:
Done